File: cs/CoreWcfDavcnaBlagajna/Models/Racun.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Threading.Tasks;

namespace CoreWcfDavcnaBlagajna.Models
{
    [DataContract(Namespace = "http://si.feri.soa/davcnablagajna/data/racun/1.0")]
    public class Racun
    {
        [DataMember(Name = "Id",Order = 1)]
        public string Id { get; set; }
        [DataMember(Name = "DavcnaSt", Order = 2)]
        public string DavcnaSt { get; set; }

        [DataMember(Name ="Datum", Order = 3)]
        public string Datum { get; set; }

        [DataMember(Name = "Znesek", Order = 4)]
        public double Znesek { get; set; }

        [DataMember(Name = "PpId", Order = 5)]
        public string PpId { get; set; }
    }
}

File: java/davcna-blagajna-soap-eksplicitno/src/main/java/si/feri/soa/davcnablagajna/data/Racun.java
package si.feri.soa.davcnablagajna.data;

import jakarta.xml.bind.annotation.XmlRootElement;
import jakarta.xml.bind.annotation.XmlType;

@XmlRootElement
@XmlType(name = "Racun",
    namespace = "http://si.feri.soa/davcnablagajna/data/racun/1.0")
public class Racun {
    private String id;
    private String davcnaSt;
    private String datum;

    public String getId() {
        return id;
    }
    public void setId(String id) {
        this.id = id;
    }
    public String getDavcnaSt() {
        return davcnaSt;
    }
    public void setDavcnaSt(String davcnaSt) {
        this.davcnaSt = davcnaSt;
    }
    public String getDatum() {
        return datum;
    }
    public void setDatum(String datum) {
        this.datum = datum;
    }
    public double getZnesek() {
        return znesek;
    }
    public void setZnesek(double znesek) {
        this.znesek = znesek;
    }
    public String getPpId() {
        return ppId;
    }
    public void setPpId(String ppId) {
        this.ppId = ppId;
    }
    private double znesek;
    private String ppId;
}
